.travis.yml

```diff
@@ -121,60 +157,59 @@ matrix:
         - ${PIP} install -r requirements.txt
       script:
         - tox
-    - name: doctest36
+
+    - name: doctest37
```

Comment:
I'm fine with moving this to 3.8 and skip over 3.7 if you don't see any problems with that.